src/node/test/merkle_bench.cpp

```diff
@@ -181,6 +183,42 @@ static void serialise_deserialise(picobench::state& s)
   s.stop_timer();
 }
 
+static void serialised_size(picobench::state& s)
+{
+  ccf::MerkleTreeHistory t;
+  vector<crypto::Sha256Hash> hashes;
+  std::random_device r;
+
+  for (size_t i = 0; i < s.iterations(); ++i)
+  {
+    crypto::Sha256Hash h;
+    for (size_t j = 0; j < crypto::Sha256Hash::SIZE; j++)
+      h.h[j] = r();
+
+    hashes.emplace_back(h);
+  }
+
+  size_t index = 0;
+
+  for (auto _ : s)
+  {
+    (void)_;
+    t.append(hashes[index++]);
+  }
+  s.start_timer();
+  auto buf = t.serialise();
+  auto bph = ((float)buf.size()) / index;
+  std::cout << fmt::format(
+                 "mt_serialize n={} : {} bytes, {} bytes/hash, {}% overhead",
+                 index,
+                 buf.size(),
+                 bph,
+                 (bph - crypto::Sha256Hash::SIZE) * 100 /
+                   crypto::Sha256Hash::SIZE)
+            << std::endl;
+  s.stop_timer();
```

Comment:
`stop_timer` should be straight after serialise. Although we may not be monitoring this timing result, that will at least make it more precise.

Reply:
Sure (but yes, we don't really care).